runtime/doc/term.txt

```diff
@@ -42,34 +43,34 @@ unlike most other environment variables.
 
   For this terminal           Set $TERM to                  |builtin-terms|
   -------------------------------------------------------------------------
+  anything libvte-based       vte, vte-256color                   Y
+   (e.g. GNOME Terminal)      (aliases: gnome, gnome-256color)
+  ConEmu                      conemu                              Y
+  cygwin                      cygwin                              Y
+  Interix                     interix                             Y
   iTerm (original)            iterm, iTerm.app                    N
   iTerm2 (new capabilities)   iterm2, iTerm2.app                  Y
   Konsole                     konsole-256color                    N
-  anything libvte-based       vte, vte-256color                   Y
-   (e.g. GNOME Terminal)      (aliases: gnome, gnome-256color)
-  tmux                        tmux, tmux-256color                 Y
-  screen                      screen, screen-256color             Y
+  Linux virtual terminal      linux, linux-256color               Y
   PuTTY                       putty, putty-256color               Y
+  rxvt                        rxvt, rxvt-256color                 Y
+  screen                      screen, screen-256color             Y
+  simple terminal (st)        st, st-256color                     Y
   Terminal.app                nsterm                              N
-  Linux virtual terminal      linux, linux-256color               Y
+  tmux                        tmux, tmux-256color                 Y
+  vtpcon                      vtpcon                              Y
+  Windows console (cmd.exe)   win32con                            Y
```

Comment:
@erw7 I could not find any info on "vtpcon". Does it have a full name? Is there a URL that describes it?

Reply:
`vtpcon` is the name I named irresponsibly, because vim calls "on MS-Windows console: support for 'termguicolors'" as `vtp`. Also, `win32con` named it because &term is `win32` on vim. "cmd.exe" also works on `ConEmu`, so I think "cmd.exe" is not appropriate. I think that it is better to say "conhost.exe" on the terminal, but what about it?

+ `vtpcon` is a "conhost.exe" that supports `ENABLE_VIRTUAL_TERMINAL_PROCESSING`
+ `win32con` is "conhost.exe" which does not support `ENABLE_VIRTUAL_TERMINAL_PROCESSING`.

Reply:
Since terminal emulation are provided by `libuv`  rather than `conhost`, `libuv` or `uvtty` may be more appropriate than `win32con`.

Reply:
> Also, win32con named it because `&term` is win32 on vim. 

@erw7 	should we use the same TERM name (`win32`) in that case?

Reply:
In the case of vim, tgc can be used when `&term=win32` and `has('vcon')`, so neovim does not behave like vim, but it may be better to change the TERM name from `win32con` to `win32` and from `vtpcon` to `vcon`.